be/src/common/config.h

```diff
@@ -260,6 +260,8 @@ CONF_Int64(index_stream_cache_capacity, "10737418240");
 
 // Cache for storage page size
 CONF_String(storage_page_cache_limit, "20G");
+// Ratio for index page cache
```

Comment:
I think it's better to give more description, such as what does the left size used for?